tachyon/math/elliptic_curves/msm/algorithms/icicle/icicle_msm.h

```diff
@@ -160,6 +161,7 @@ bool IcicleMSM<bn254::G1AffinePoint>::Run(const BaseContainer& cpu_bases,
 #if FIELD_ID != BN254
 #error Only Bn254 is supported
 #endif
+  TRACE_EVENT("MSM", "Icicle::Tachyon::BN254::G1::MSM");
```

Comment:
Why the trace format is different from ICICLE NTT, how about below?

```suggestion
  TRACE_EVENT("MSM", "Icicle::MSM");
```